lib/localio/processors/csv_processor.rb

```diff
@@ -56,7 +56,13 @@ def self.load_localizables(platform_options, options, allowed_languages)
     first_term_row = first_valid_row_index+1
     last_term_row = last_valid_row_index-1
 
+    application_index = csv_file[first_valid_row_index].index('Application')
```

Comment:
This could be omitted for cases where platform_name isn't specified, but it doesn't seem like a bad sanity check to have regardless.